brave/brave-core
# Add some ad diagnostics to rewards internal page.

Following diagnostics added:
- Users locale [LocaleHelper]
- Status: Enabled/disabled [prefs::kEnabled]
- Catalog id [prefs::kCatalogId]
- Catalog last updated timestamp [prefs::kCatalogLastUpdated]
- Last un-idle timestamp [AdsImpl::OnUnIdle]

![image](https://user-images.githubusercontent.com/86598290/127129479-23953ece-6595-4d2c-915f-238aa4245bb4.png)

![image](https://user-images.githubusercontent.com/86598290/127131272-f4e24462-7d87-4511-9864-fddce807778d.png)




Resolves https://github.com/brave/brave-browser/issues/10964

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1) Check that `Ad diagnostics` tab is shown on rewards-internals page
2) Check that following diagnostics are shown:
  - Status: Enabled/disabled
  - Users locale
  - Catalog id
  - Catalog last updated timestamp 
  - Last un-idle timestamp
